packages/eslint-plugin/src/rules/no-deprecated.ts

```diff
@@ -232,7 +266,7 @@ export default createRule<Options, MessageIds>({
       }
     }
 
-    function getCallLikeNode(node: TSESTree.Node): CallLikeNode | undefined {
+    function getCallLikeNode(node: TSESTree.Node): CalleeNode | undefined {
```

Comment:
[Naming] If the type's name is switched from `CalleeNode` to `CallLikeNode`, the corresponding `get*` function's name should too:

```suggestion
    function getCalleNode(node: TSESTree.Node): CalleeNode | undefined {
```